app/src/index.ts

```diff
@@ -11,7 +11,7 @@ import { pages } from './screens/OnboardingPages'
 import Splash from './screens/Splash'
 import Terms from './screens/Terms'
 import { defaultTheme as theme } from './theme'
-const localization = _merge({}, translationResources, {
+const localization = merge({}, translationResources, {
```

Comment:
Not overly important but a space above here may help readability.

Reply:
The linter should automatically take care of formatting here. Not sure if it's not working?